src/sentry/static/sentry/app/components/alert.jsx

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import {cx} from 'emotion';
 import styled from 'react-emotion';
 import isPropValid from '@emotion/is-prop-valid';
+import Color from 'color';
```

Comment:
What is this `color` package? I don't see it in `package.json`

Reply:
Also should it be imported as `Color` and not `color`?

Reply:
sorry—I must have dumped my changes to `package.json`. This also probably explains why it isn't passing builds.

Reply:
yes. they capitalize it in the docs but the [whole thing is just a function](
https://github.com/Qix-/color/blob/master/index.js) so I don't know why.